five.js

```diff
@@ -81,7 +81,7 @@
   five.smooth = function() { return 'S'; };
 
   five.tooSlow = function() {
-    var returnIn = new Date(new Date().valueOf() + 555);
+    var returnIn = new Date(new Date().valueOf() + parseInt("" + this() + this() + this()));
```

Comment:
You can actually make this a bit nicer :) `five` has it's `valueOf` property set which means we can change your code to 
`'' + five + five + five` :)